tests-integration/tests/http.rs

```diff
@@ -436,6 +436,16 @@ pub async fn test_sql_api(store_type: StorageType) {
     // this is something only json format can show
     assert!(format!("{:?}", output[0]).contains("\\\"param\\\""));
 
+    // test csv format
+    let res = client
+        .get("/v1/sql?format=csv&sql=select cpu,ts,host from demo limit 1")
+        .send()
+        .await;
+    assert_eq!(res.status(), StatusCode::OK);
+    let body = &res.text().await;
+    // Must be escaped correctly: 66.6,0,"host, ""name"
```

Comment:
[RFC-4180](https://www.ietf.org/rfc/rfc4180.txt) says "If double-quotes are used to enclose fields, then a double-quote appearing inside a field must be escaped by preceding it with another double quote."